docs/deployment/landing_page.rst

```diff
@@ -249,3 +249,77 @@ that user and group file permissions are locked down and that modules or
 gateway interfaces for dynamic scripting languages are not enabled. You
 don't want any unnecessary code or services running as this increases
 the attack surface.
+
+Landing page content suggestions
+--------------------------------
+
+The content below presents sample text for the SecureDrop component of a news 
+organization’s tips page. It does not account for any specific legal 
+or organizational needs, but should provide guidance for any outlet getting 
+started on crafting landing page language. Any tweaks to the sample content 
+should be left to the legal and editorial discretion of the individual outlet, 
+and should be viewed as essential to upholding source protection and transparency.
+
+**What is SecureDrop?**
+
+SecureDrop is an anonymity tool for journalists and whistleblowers. As a source, 
+you can use our SecureDrop installation to anonymously submit documents to our 
+organization. Our journalists use SecureDrop to receive source materials and 
+securely communicate with anonymous contacts.
+
+**What should I know before submitting material through SecureDrop?**
+
+To protect your anonymity when using SecureDrop, it is essential that you do 
+not use a network or device that can easily be traced back to your real 
+identity. Instead, use public wifi networks and devices you control.
+
+- Do NOT access SecureDrop on your employer’s network.
+
+- Do NOT access SecureDrop using your employer’s hardware. 
+
+- Do NOT access SecureDrop on your home network. 
+
+- DO access SecureDrop on a network not associated with you, like the wifi at a library or cafe.
+
+**Got it. How can I submit files and messages through SecureDrop?**
+
+Once you are connected to a public network at a cafe or library, download 
+and install the `Tor Browser bundle <https://www.torproject.org/projects/torbrowser>`_. 
+
+Launch the Tor Browser. Visit our organization’s unique SecureDrop URL 
+at our unique URL. Follow the instructions you find on our source page to 
+send us materials and messages.
+
+When you make your first submission, you will receive a unique codename. 
+Memorize it. If you write it down, be sure to destroy the copy as soon as 
+you’ve committed it to memory. Use your codename to sign back in to 
+our source page, check for responses from our journalists, and upload 
+additional materials.
+
+**As a source, what else should I know?**
+
+No tool can absolutely guarantee your security or anonymity. 
+The best way to protect your privacy and anonymity as a source 
+is to adhere to best practices.
+
+You can use a separate computer you’ve designated specifically to handle 
+the submission process. 
+Or, you can use an alternate operating system like Tails, 
+which boots from a USB stick and erases your activity at the end of every session.
+
+A file contains valuable metadata about its source — when it was created 
+and downloaded, what machine was involved, the machine’s owner, etc.
```

Comment:
Metadata could also include watermarks

Reply:
Good point. Happy to add that facet of document metadata. "Etc." is meant to point to the endless sources and manifestations of metadata, however.